rules/kfddistribution-kfd-v1alpha2.yaml

```diff
@@ -52,3 +52,15 @@ distribution:
     reducers:
       - key: distributionModulesTracingTempoBackend
         lifecycle: pre-apply
+  - path: .spec.distribution.modules.dr.type
+    immutable: false
+    description: "changes to the dr module type have been detected. This will cause the replacement of the current DR (velero) stack (removal or creation)."
```

Comment:
```suggestion
    description: "changes to the Disaster Recovery module type have been detected. This will cause the replacement of the current DR (velero) stack (removal or creation)."
```